content/reference-ledger-format.md

```diff
@@ -398,3 +398,72 @@ The `index` of a RippleState node is the SHA-512Half of the following values put
+                "SignerWeight": 1
+            }
+        }
+    ],
+    "SignerListID": 0,
+    "SignerQuorum": 3,
+    "index": "A9C28A28B85CD533217F5C0A0C7767666B093FA58A0F2D80026FCC4CD932DDC7"
+}
+```
+
+A SignerList node has the following fields:
+
+| Name            | JSON Type | Internal Type | Description |
+|-----------------|-----------|---------------|-------------|
+| OwnerNode       | String    | UInt64        | A hint indicating which page of the owner directory links to this node, in case the directory consists of multiple nodes. |
+| SignerQuorum    | Number    | UInt32        | A target number for signer weights. To produce a valid signature for the owner of this SignerList, the signers must provide valid signatures whose weights sum to this value or more. |
+| SignerEntries   | Array     | Array         | An array of SignerEntry objects representing the parties who are part of this signer list. |
+| SignerListID    | Number    | UInt32        | An ID for this signer list. Currently always set to `0`. If a future update allows multiple signer lists for an account, this may change. |
+| PreviousTxnID   | String    | Hash256       | The identifying hash of the transaction that most recently modified this node. |
+| PreviousTxnLgrSeq | Number  | UInt32        | The sequence number (`ledger_index`) of the ledger that contains the transaction that most recently modified this node. |
+
+The `SignerEntries` may be any combination of funded and unfunded addresses that use either secp256k1 or ed25519 keys.
+
+### SignerEntry object ###
+
+Each member of the `SignerEntries` field is an object that describes that signer in the list. A SignerEntry has the following fields:
+
+| Name            | JSON Type | Internal Type | Description |
+|-----------------|-----------|---------------|-------------|
+| Account         | String    | AccountID     | An address whose signature contributes to the multi-signature. This does not need to be a funded Ripple account. |
+| SignerWeight    | Number    | UInt16        | The weight of signatures from this signer. A multi-signature is only valid of the sum weight of the signatures provided meets or exceeds the SignerList's `SignerQuorum` value. |
+
+When processing a multi-signed transaction, the server dereferences the `Account` values with respect to the ledger at the time of transaction execution. If the address _does not_ correspond to a funded [AccountRoot node](#accountroot), then only the master secret associated with that address can be used to produce a valid signature. If the account _does_ exist in the ledger, then it depends on the state of that account. If the account has a Regular Key configured, the Regular Key can be used. The account's master key can only be used if it is not disabled. Even if the account has a SignerList configured, a multi-signature cannot be used as a valid component to another multi-signature. (In other words, "multi-level" multi-signing is disallowed.)
```

Comment:
I would be inclined to omit the last two sentences.  Yeah, I implemented 2-level multi-sign and then threw it out.  But it might be best if we don't even suggest that multi-level multi-sign is a consideration at this point.